contrib/cirrus/lib.sh

```diff
@@ -244,6 +244,11 @@ use_netavark() {
     export NETWORK_BACKEND=netavark  # needed for install_test_configs()
     msg "Removing any/all CNI configuration"
     rm -rvf /etc/cni/net.d/*
+
+    # TODO: Remove this when netavark/aardvark-dns development slows down
+    warn "Updating netavark/aardvark-dns to avoid frequent VM image rebuilds"
+    # N/B: This is coming from updates-testing repo in F36
+    lilto dnf update -y netavark aardvark-dns
```

Comment:
Should these be installed from updates-testing?

Reply:
The comment suggests that they are, but the string `updates.test` does not appear  anywhere else (relevantly) in this repo, nor in containers-automation, nor in containers-image, so I have this question too

Reply:
Code that does it is here: https://github.com/containers/automation_images/pull/115/files#diff-66757a985486a838bd165fde75e7f0d50de7281ceab5e044cd7625979956e65cR28-R34

log of it being done is here: https://cirrus-ci.com/task/5514692318724096?logs=main#L41

Reply:
So updates-testing repo is enabled.